### DISCUSSION TITLE
[UI/UX] Reports repeat currency all over

### DISCUSSION THREAD

**Discussion by beanaroo** [firefly-iii/firefly-iii#4809] [Firefly III] [answered]
I have really been enjoying Firefly III this year.

One usability concern I have is the way a Category Report will repeat the currency on _every_ row in _every_ column.

In the screenshot below:
- `New Zealand Dollar` is repeated 25 times
- `NZ$` is repeated 75 times
- With more on the rest of the report

![firefly-category-report](https://user-images.githubusercontent.com/1748909/120064792-3593d000-c0c2-11eb-92b2-6abd44f9da8a.png)

- I'd like to understand if this is a known problem and if it's worth opening an issue.
- If all accounts and categories are displayed in the same currency, could this not be better indicated in the report/table titles and column headers?
- In the event multiple currencies are present, how would the `Sum` row work?

**Comment by JC5** [comment, marked as answer]
I thought that the report would ignore the currency if there was only one. I'll pick it up in the new layout.

If there are multiple currencies, they will be summed up separately.

**Reply by beanaroo** [reply]
Ah, that's awesome! Thank you for clarifying.

I should mention, I do have multiple currencies enabled as I have some transactions with Foreign Amounts recorded
